lib/v4/errorMap/errorMap.ts

```diff
@@ -0,0 +1,126 @@
+import { joinPath } from '../../utils/joinPath.ts';
+import { isNonEmptyArray } from '../../utils/NonEmptyArray.ts';
+import { titleCase } from '../../utils/titleCase.ts';
+import { parseInvalidElementIssue } from './invalidElement.ts';
+import { parseInvalidStringFormatIssue } from './invalidStringFormat.ts';
+import { parseInvalidTypeIssue } from './invalidType.ts';
+import { parseInvalidValueIssue } from './invalidValue.ts';
+import { parseNotMultipleOfIssue } from './notMultipleOf.ts';
+import { parseTooBigIssue } from './tooBig.ts';
+import { parseTooSmallIssue } from './tooSmall.ts';
+import { parseUnrecognizedKeysIssue } from './unrecognizedKeys.ts';
+import { parseInvalidKeyIssue } from './invalidKey.ts';
+import { parseCustomIssue } from './custom.ts';
+import type {
+  IssueType,
+  ErrorMapOptions,
+  AbstractSyntaxTree,
+} from './types.ts';
+import type * as zod from 'zod/v4/core';
+
+export const issueParsers: Record<
+  IssueType,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (issue: any, options: ErrorMapOptions) => AbstractSyntaxTree
+> = {
```

Comment:
praise: well thought adding this as `Record` so that the code is future-proof and types complain in case a new error is added, or an existing one is removed 👌